src/pages/Settings/Settings.js

```diff
@@ -19,25 +17,27 @@ export default function Settings() {
       <Header>
         <span className="l-content settings__header">Settings</span>
       </Header>
-      <div className="l-content">
+      <div className="l-content settings">
         <div className="settings__toggles">
-          <Notification>
-            You will need to refresh your browser for this change to take effect
-          </Notification>
-          <label className="row">
-            <div className="col-6">Disable Analytics</div>
-            <div className="col-3">
+          <div className="settings__toggles-group">
+            <label
+              onChange={() => {
```

Comment:
I'm not sure why this was moved to the label?

Reply:
This was moved as clicking on the wrapper label toggles the checkbox.. however, it probably better if it's on the input, as it was originally - will revert.